packages/rspack/tests/configCases/builtin-swc-loader/styled-components/index.jsx

```diff
@@ -2,11 +2,9 @@ import { Container } from "./Button";
 import { Container as Container2 } from "./Button2";
 
 it("styled components", () => {
-	expect(Container.displayName).toMatch("Button__Container");
```

Comment:
Why was it removed.

Reply:
Webpack test runner is running in mode production by default in which it does not have this field.

Reply:
Webpack test runner is running in mode production by default in which it does not have this field.